python/paddle/fluid/tests/unittests/op_test.py

```diff
@@ -223,7 +225,7 @@ def _append_ops(self, block):
 
     def _get_io_vars(self, block, numpy_inputs):
         inputs = {}
-        for name, value in numpy_inputs.iteritems():
+        for name, value in numpy_inputs.items():
```

Comment:
add a todo to make them six.iteritems?

Reply:
TODO has been added [here](https://github.com/PaddlePaddle/Paddle/pull/12390/commits/d8ddd3b3078ebf5ce7c374b78f2bfd598ae38f49#r206030485)